sdks/core/src/main/java/org/gbif/pipelines/core/parsers/temporal/TemporalParser.java

```diff
@@ -209,4 +209,34 @@ protected static boolean isValidDate(
 
     return likelyRange.contains(LocalDate.of(year, month, day));
   }
+
+  public static Optional<Long> getTime(TemporalAccessor temporalAccessor) {
+    if (temporalAccessor == null) {
+      return Optional.empty();
+    }
+
+    int year;
+    if (temporalAccessor.isSupported(ChronoField.YEAR)) {
```

Comment:
Are these changes intended to be in this commit?

Reply:
OK - it was just the reference to `ChronoField` that jumped out as chrono isn't used in the [identification history extension](https://rs.gbif.org/extension/dwc/identification_history_2022-02-02.xml). Made me just wonder if that was unrelated work slipping in

Reply:
Ah it's because I use the `dateIdentified ` field of the extension for the sorting and since it's a string I have to parse it and convert it into a date.